pyproject.toml

```diff
@@ -0,0 +1,56 @@
+[build-system]
+requires = ["setuptools >= 61"]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "compare-locales"
+description = "Lint Mozilla localizations"
+readme = "README.md"
+license = { text = "MPL-2.0" }
+authors = [
+  { name = "Mozilla", email = "l10n-drivers@mozilla.org" },
+  { name = "Axel Hecht" },
+  { name = "Eemeli Aro", email = "eemeli@mozilla.com" },
+]
+classifiers = [
+  "Development Status :: 5 - Production/Stable",
+  "Intended Audience :: Developers",
+  "License :: OSI Approved :: Mozilla Public License 2.0 (MPL 2.0)",
+  "Operating System :: OS Independent",
+  "Programming Language :: Python",
+  "Programming Language :: Python :: 3",
+  "Programming Language :: Python :: 3.7",
+  "Programming Language :: Python :: 3.8",
+  "Programming Language :: Python :: 3.9",
```

Comment:
Add 3.10, given we're testing it now and it's in tox.ini?

Reply:
D'oh, of course.

```suggestion
  "Programming Language :: Python :: 3.9",
  "Programming Language :: Python :: 3.10",
```